@l10n/ja/docs/use-cases/tokenization/authorized-minter.md

```diff
@@ -47,15 +47,15 @@ NFTの取引は、売りオファーの作成または買いオファーの承
 
 NFTをオークション形式で販売することができます。[NFTオークションの実行](../../concepts/tokens/nfts/running-an-nft-auction.md)をご覧ください。
 
-あなたはブローカーとして、売り手と入札者をつなぎ、取引を完了させ、購入価格の何パーセントかを保持することができます。[NFTokenの取引を仲介する](../../tutorials/javascript/nfts/broker-an-nft-sale.md)をご覧ください。
+あなたはブローカーとして、売り手と入札者をつなぎ、取引を完了させ、販売価格の一部を報酬として得ることもできます。[NFTokenの取引を仲介する](../../tutorials/javascript/nfts/broker-an-nft-sale.md)をご覧ください。
 
 ### 準備金要件
 
-販売用のNFTをミントする際には、XRPの準備金が必要となります。各NFTokenページには、2XRPの準備金が必要です。NFTokenページは16～32個のNFTを保管することができます。
+販売用のNFTをミントする際には、XRPの準備金が必要となります。各NFTokenページには、{% $env.PUBLIC_OWNER_RESERVE %}の準備金が必要です。NFTokenページは16～32個のNFTを保管することができます。
 
-各`NFTokenOffer`オブジェクトは、2XRPの準備金が必要です。
+各`NFTokenOffer`オブジェクトは、{% $env.PUBLIC_OWNER_RESERVE %}の準備金が必要です。
 
-`NFTokenOffer`を作成したり、NFTを売却したりする際には、些細な送金手数料（およそ6000ドロップ、または0.006 XRP）が発生します。大量に販売する場合、こうした少額の手数料はすぐにかさみますので、ビジネスのコストとして考慮する必要があります。
+`NFTokenOffer`を作成したり、NFTを売却したりする際には、わずかな送金手数料（およそ6000ドロップ、または0.006 XRP）が発生します。大量に販売する場合、こうした少額の手数料はすぐにかさみますので、ビジネスのコストとして考慮する必要があります。
```

Comment:
@mDuo13 
The same applies to the English version, but do you know where the fees for 6000drops come from?

Reply:
I'm not sure either. I've asked @DennisDawson if he knows.